### DISCUSSION TITLE
Localhost not connecting

### DISCUSSION THREAD

**Discussion by ArijitBh** [KRTirtho/spotube#188] [Q&A]
Hi. I can't seem to complete the verification of the API. The localhost is throwing an Err_server_timeout or err_connection_refused.

**Comment by KRTirtho** [comment]
Let me guess you're using Lineage OS or a Samsung device?

**Reply by Divine-Bovine** [reply]
So.. I'm facing this issue as well with the version 2.4.1
I'm trying to login to Spotify through Spotube, however, everytime the localhost seems to be unable to connect and the request times out.

Version: 2.4.1
Spotify Development App: Created
Status: Development Mode
Redirect URI: http://localhost:4304/auth/spotify/callback
Phone: Samsung - Android 12 (Non-rooted).

@KRTirtho - Can you please advise on what could be the cause for this?

**Reply by KRTirtho** [reply]
I need to see what's going on...
Can provide me the logs file located in `Android/data/oss.krtirtho.spotube/files/.spotube_logs`
Since you're using Android 12, you'll need [Solid Explorer](https://play.google.com/store/apps/details?id=pl.solidexplorer2) to access `Android/data` folder

**Reply by Divine-Bovine** [reply]
I cleared everything and restarted the process. The logs for today's instance are attached.

[Spotube Logs.log](https://github.com/KRTirtho/spotube/files/9564510/Spotube.Logs.log)

**Reply by KRTirtho** [reply]
Can you check if the latest nightly fixes your issue
https://spotube.netlify.app/other-downloads/nightly-downloads

**Reply by Divine-Bovine** [reply]
No change. Still the same error.

**Reply by KRTirtho** [reply]
> I need to see what's going on... Can provide me the logs file located in `Android/data/oss.krtirtho.spotube/files/.spotube_logs` Since you're using Android 12, you'll need [Solid Explorer](https://play.google.com/store/apps/details?id=pl.solidexplorer2) to access `Android/data` folder

Can you do this again?

**Reply by Divine-Bovine** [reply]
Hey.. I still am unable to login with Spotify but did succeed in getting the "Authorization successful, return to Spotube" message. After that message, Spotube doesn't change screens at all or the 'Done' option in the step by step guide doesn't get enabled.

The authorization successful messages comes up when I try to login to Spotify while a song is playing. It weirdly doesn't work when nothing is playing. Returning back to Spotube doesn't seem to help as no screen changes.

Attached is the log file as requested.
[spotube_logs Oct 2nd.txt](https://github.com/KRTirtho/spotube/files/9692444/spotube_logs.Oct.2nd.txt)

**Reply by Divine-Bovine** [reply]
@KRTirtho - After multiple tries, while a song was playing, it finally worked and it connected to my Spotify account in the same nightly version.

**Comment by ArijitBh** [comment]
Nope. Vinalla unmodded android on Moto G10 Power. Dev options turned off.

**Comment by KRTirtho** [comment]
@ArijitBh and @Divine-Bovine see https://github.com/KRTirtho/spotube/issues/152#issuecomment-1265625104
